### PR TITLE
updates docs for Halyard PR #733

### PR DESCRIPTION
**Pull request by omawhite**
see [PR](https://github.com/spinnaker/halyard/pull/733)

### CODE REVIEW

**Comment by omawhite** [comment]
@lwander

File: reference/halyard/custom.md [hunk 1]
```diff
@@ -76,14 +76,27 @@ cluster.
...
 | `overrideBaseUrl` | The baseURL this service is reachable on. This is already made configurable for Deck & Gate via `hal config security`, since these are both public-facing and may service from different hosts than they are discoverable on internal to Spinnaker. |
 | `port` | The port number this service is bound to, and will accept requests on. |
 | `safeToUpdate` | Whether or not this service can be shutdown, and spun on a new VM/container. This protects datastores like Vault & Redis from being taken down from Halyard. |
-| `scheme` | The URI scheme used to addres this service, e.g. `http` vs.  `https`. |
+| `scheme` | The URI scheme used to address this service, e.g. `http` vs.  `https`. |
 | `targetSize` | The initial number of nodes this service will be created with.  This is only respected on the initial deployment, and further edits will be rejected in favor of the prior service version's size. |
+
+
+### Kubernetes
+The following Kubernetes specific setting can be specified
```
Comment (lwander) [inline reference/halyard/custom.md]: Please insert newlines after the headers - some markdown formatting requires it.

Comment (lwander) [inline reference/halyard/custom.md]: Please motivate why a user would want to do this

**Review by lwander — APPROVED** [review]

File: reference/halyard/custom.md (hunk 1 above)
Comment (edwinavalos) [reply reference/halyard/custom.md]: Something along these lines.

"If making use of custom artifactIds from an authenticated docker registry imagePullSecrets must be made available in the replicaset definition. To specify imagePullSecrets to custom artifactIds they can be specified as follows:"

**Pull request merged**